src/Components/CreateAccount.jsx

```diff
@@ -97,6 +97,10 @@ class CreateAccount extends Component {
     }
   }
 
+  isStaff() {
+    return ((this.state.role === "Teacher") || (this.state.role === "teacher")) ? true : false;
```

Comment:
if you are going to do this we should compare using localeCompare with a sensitivity set so it just compares the strings case insensitive